sdk/purview/azure-analytics-purview-sharing/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Remove this duplicated section?

Reply:
Thanks.  This should be addressed in the latest change.